trin-history/src/jsonrpc.rs

```diff
@@ -244,31 +248,9 @@ impl HistoryRequestHandler {
             return Ok(Value::String(hex::encode(content)));
         }
 
-        // Construct trace response
-        let closest_nodes: Vec<NodeInfo> = closest_nodes
-            .iter()
-            // Skip over enrs that are unable to be looked up
-            .filter_map(|node_id| {
-                let content_id = content_key.content_id();
-                // Return None for enrs that cannot be looked up
-                match self.network.overlay.discovery.find_enr(node_id) {
-                    Some(enr) => XorMetric::distance(&content_id, &enr.node_id().raw())
-                        .log2()
-                        .map(|distance| {
-                            let distance = Distance::from(distance);
-                            NodeInfo {
-                                enr: EthEnr::from_str(&enr.to_base64()).unwrap(),
-                                distance,
-                            }
-                        }),
-                    None => None,
-                }
-            })
-            .collect();
-
-        Ok(json!(TraceContentInfo {
-            content: content.into(),
-            route: closest_nodes,
+        Ok(json!({
```

Comment:
I think it is more clear if we return the `TraceContentInfo` type, instead of json map.